lang/java/avro/pom.xml

```diff
@@ -55,7 +55,42 @@
         </includes>
       </resource>
     </resources>
+    <testResources>
+      <testResource>
+        <directory>src/test/resources</directory>
+      </testResource>
+      <testResource>
+        <directory>../../../share/</directory>
+        <includes>
+          <include>schemas/**</include>
+          <include>test/**</include>
+        </includes>
+        <targetPath>share/</targetPath>
+      </testResource>
+    </testResources>
+
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-toolchains-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>Build using JDK 21</id>
+            <phase>validate</phase>
+            <goals>
+              <goal>toolchain</goal>
+            </goals>
+            <configuration>
+              <toolchains>
+                <jdk>
+                  <version>21</version>
+                </jdk>
+              </toolchains>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
```

Comment:
The toolchain plugin is also copied into the modules. Is that needed? Can't it inherit from here?

Reply:
If all of the modules build with the same jdk it can.

Reply:
Originally I had different JDK versions for various modules. Mainly because of the Unsafe ... which I eventually kicked.